### DISCUSSION TITLE
About the bug of tool rsa-key-pair-generator

### DISCUSSION THREAD

**Discussion by harvey2953** [he3-app/he3-feedback#160] [Bug Report]
### Version
rsa-key-pair-generator 1.0.0

### Environment
Mac v1.3.2

### What is expected?
public key title error

### What is actually happening?
the title both Private Key

**Comment by congcongnalian** [comment]
非常高兴您能提出反馈！该问题已经修复，点击最新链接查看：https://t.he3app.com?sa7y
![image](https://user-images.githubusercontent.com/68501045/237058310-c3cab151-9180-46de-83fd-a592923e4a5a.png)
